openwisp_monitoring/monitoring/configuration.py

```diff
@@ -206,7 +206,9 @@ def _get_access_tech():
                     _('Total download traffic'),
                     _('Total upload traffic'),
                 ],
-                'unit': _(' GB'),
+                'conversion': {
+                    'filter': '{unit}',
```

Comment:
this name kind of puzzles me, why did you choose `{unit}`?

Reply:
> Here's another thing which I have noticed during my testing:
> 
> ![Screenshot from 2022-06-21 23-52-18](https://user-images.githubusercontent.com/841044/174939919-eb850651-5acd-4a6f-813d-f9f155626f5a.png)
> 
> I would expect to see MB in the summary in a case like this.

yeah! I did not apply those conditions for percircles, but I would change.

Reply:
I choose `{unit}` because like the conversion happening in values would pass a specific unit either of these `GB, MB, KB`. If you suggest something else I will update it.

Reply:
Yeah! as an average of the data would have been less than `1 GB`, that's why it would show all data in `MB`, basically where the points which have no data are also calculated in the average that is why the average has been lowered and the data it converts shown to be in  `MB`.

Reply:
> Good progress, for the moment the most obvious points to improve I see are the following, but first, let's look at the screenshot:
> 
> ![yashika-1](https://user-images.githubusercontent.com/841044/174913986-49d62590-ffc1-4230-8fca-9657808ff476.png)
> 
> 1. there should be a space between the number and the unit, so 20GB > 20 GB
> 2. in the example I'm showing, the unit chosen is MB but I think GB would be more suited, what do you think?
> 
> Regarding the code, I think we can see this in our next meeting, but for the moment @atb00ker lert's focus on the functionality, let's make it work as we need first, that's the most important thing, code quality comes later.


Yeah! as an average of the data would have been less than `1 GB`, that's why it would show all data in `MB`, basically where the points which have no data are also calculated in the average that is why the average has been lowered and the data it converts shown to be in  `MB`.

Reply:
what if we change this to:

```
'unit': 'adaptive_bytes`,
```

Then just pass `{unit}` where the code needs this, if at all.